### PR TITLE
SSCS-5994 - Set furtherEvidenceReceived on the furtherEvidenceAction

### PR DESCRIPTION
**Pull request by ryandac**
Set furtherEvidenceReceived on the furtherEvidenceAction field (for DWP) when FE has been bulk scanned

https://tools.hmcts.net/jira/browse/SSCS-5994

commons PR : https://github.com/hmcts/sscs-common/pull/197

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionfurtherevidence/ActionFurtherEvidenceReceivedHandler.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+
+@Service
+public class ActionFurtherEvidenceReceivedHandler implements PreSubmitCallbackHandler<SscsCaseData> {
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionfurtherevidence/ActionFurtherEvidenceReceivedHandler.java]: I find all these class names relating to the action further evidence event quite confusing now. Could we agree a pattern and try and use this? Happy to discuss...

File: src/main/resources/config/application.properties [hunk 2]
```diff
@@ -30,7 +30,7 @@ subscriptions.mac.secret=${SUBSCRIPTIONS_MAC_SECRET:our-big-secret}
 
 spring.application.name=TribunalsCaseApi
 
-feature.issue_further_evidence=${ISSUE_FURTHER_EVIDENCE_ENABLED:false}
+feature.issue_further_evidence=${ISSUE_FURTHER_EVIDENCE_ENABLED:true}
```
Comment (jackmaloney) [inline src/main/resources/config/application.properties]: Should this be true? This was not meant to be switched on until end of August

Comment (ryandac) [reply src/main/resources/config/application.properties]: It's off in demo, aat and prod. They can be switched on at that point in those environments.

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/InterlocServiceHandler.java [hunk 3]
```diff
@@ -20,7 +20,6 @@ public InterlocServiceHandler() {
         eventTypeToSecondaryStatus.put(EventType.INTERLOC_SEND_TO_TCW, "reviewByTcw");
         eventTypeToSecondaryStatus.put(EventType.TCW_DIRECTION_ISSUED, "awaitingInformation");
         eventTypeToSecondaryStatus.put(EventType.INTERLOC_INFORMATION_RECEIVED, "interlocutoryReview");
-        eventTypeToSecondaryStatus.put(EventType.INTERLOC_SENT_TO_JUDGE, "reviewByJudge");
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/InterlocServiceHandler.java]: Why has this event been removed?

**Pull request merged**